plugin/deepness/dialogs/packages_installer/packages_installer_dialog.py

```diff
@@ -68,28 +82,47 @@ def __init__(self, iface, parent=None):
         self.aborted = False
         self.thread = None
 
+    def move_to_top(self):
+        """ Move the window to the top.
+        Although if installed from plugin manager, the plugin manager will move itself to the top anyway.
+        """
+        self.setWindowState((self.windowState() & ~QtCore.Qt.WindowMinimized) | QtCore.Qt.WindowActive)
+
+        if sys.platform == "linux" or sys.platform == "linux2":
```

Comment:
Was there any way to verify this? I was running macOS in a virtual machine the other day and can re-run it, but it won't let me check on the M1 architecture.

Reply:
Would be good to check, at least on the systems we can. Testing before release should be enough